easybuild/easyconfigs/c/Clang/Clang-9.0.1-check_size_and_offset.patch

```diff
@@ -0,0 +1,11 @@
+--- llvm-9.0.1.src/projects/compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cc.orig	2020-03-11 20:35:37.099808821 +0100
```

Comment:
@alexandermichels Please add a short description on top of the patch to describe what it does, why it's needed, and/or which errors it's fixing exactly.

Also mention author or add a reference to upstream fix.

Do take into account that this implies also changing the SHA256 checksums for this patch file

Reply:
@boegel I added a short description to the patch file.

@branfosj I can test out the patch you linked.